fenix/app/metrics.yaml

```diff
@@ -9140,6 +9140,104 @@ awesomebar:
     metadata:
       tags:
         - Search
+  sponsored_suggestion_clicked:
+    type: event
+    description: |
+      A sponsored suggestion in the awesomebar was clicked.
+    bugs:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1871156
+    data_reviews:
+      - TODO
+    data_sensitivity:
+      - interaction
+    notification_emails:
+      - android-probes@mozilla.com
+      - lina@mozilla.com
+      - ttran@mozilla.com
+      - najiang@mozilla.com
+    expires: never
+    extra_keys:
```

Comment:
These extra keys are in a format that I don't recognize.  [This](https://mozilla.github.io/glean/book/reference/metrics/event.html?highlight=extra_keys%3A#extra-metric-parameters) is where Glean specify the format for extra keys.  Please make sure you test (Or have it QA'd to make sure these extra keys are working correctly)

Reply:
Hey @rocketsroger, I will be answering review comments as Lina is off :)

So those are YAML syntaxes for "Anchors and Aliases" which allow us to reuse nodes (i.e. blocks) in a config file. You can see the YAML specs [here](https://yaml.org/spec/1.2.2/#3222-anchors-and-aliases) and some examples [here](https://support.atlassian.com/bitbucket-cloud/docs/yaml-anchors/). As they're part of the YAML specs, all the mainstream YAML parsers should be able to handle them properly. We use them extensively to keep the YAML configuration files DRY :)

Reply:
Yes, I understand that it's not breaking anything.  However, for maintainability, it is better if we all follow the same format when adding new telemetry.  

The second part of the comment about QA testing was not related to the format.  I just wanted to remind Lina to have it tested by QA after landing.

Reply:
> Yes, I understand that it's not breaking anything. However, for maintainability, it is better if we all follow the same format when adding new telemetry.

We use them a lot for desktop Glean metrics, e.g. [here](https://searchfox.org/mozilla-central/source/browser/components/newtab/metrics.yaml#348-352). Is there any particular reason for avoiding those here? The main benefit of using anchors is that for those repetitive blocks, we'd only need to maintain one copy rather than duplicate it many times.

>  The second part of the comment about QA testing was not related to the format. I just wanted to remind Lina to have it tested by QA after landing.

👍

Reply:
> We use them a lot for desktop Glean metrics, e.g. [here](https://searchfox.org/mozilla-central/source/browser/components/newtab/metrics.yaml#348-352). Is there any particular reason for avoiding those here? The main benefit of using anchors is that for those repetitive blocks, we'd only need to maintain one copy rather than duplicate it many times.

I see. Maybe I'm thinking about this the wrong way.  I do want consistency but readability is better with anchors.  Maybe updating the whole file to use anchors is a good idea.

I withdraw my objection, let's land with anchors and I'll talk to the team to see if we should update the whole file after.  Thanks